python/tests/mlir/ast_break.py

```diff
@@ -34,53 +34,58 @@ def kernel(x: float):
 # CHECK-LABEL:   func.func @__nvqpp__mlirgen__kernel(
 # CHECK-SAME:                                        %[[VAL_0:.*]]: f64) attributes {"cudaq-entrypoint"} {
 # CHECK:           %[[VAL_1:.*]] = arith.constant 5.000000e+00 : f64
-# CHECK:           %[[VAL_2:.*]] = arith.constant 10 : i64
-# CHECK:           %[[VAL_3:.*]] = arith.constant 2 : i64
-# CHECK:           %[[VAL_4:.*]] = arith.constant 1 : i64
+# CHECK:           %[[VAL_2:.*]] = arith.constant 1 : i64
+# CHECK:           %[[VAL_3:.*]] = arith.constant 10 : i64
```

Comment:
ordering here may not be deterministic - might check with Eric on if we need `CHECK-DAG:`

Reply:
Addressed in 40ab1cc